pages/mesosphere/dcos/services/edge-lb/1.5/concepts/self-service/index.md

```diff
@@ -0,0 +1,327 @@
+---
+layout: layout.pug
+navigationTitle: Self Service Auto Pools
+title: Self Service Auto Pools
+menuWeight: 1
+excerpt: Describes how you can automaticly expose applications with task labels
+enterprise: true
+---
+
+Edge-LB supports automatic provisioning of the pools based on the Marathon app labels / SDK tasks labels (annotations). Pool configuration is driven from a pool template. A default pool template is created at the time of Edge-LB installation. The template is rendered into a pool configuration based on the values of the app / task [labels](#supported-labels) (discussed below).
```

Comment:
I'd would add line explaining the value instead of focusing on the technicalities. Example:

```md
Self Service Auto Pools streamlines deployment by enabling developers to _self-service_  [pools]( `architecture/#edge-lb-pool`) using Marathon app labels or SDK task labels (annotations).
Edge-LB automatically provisions pools using pool templates. A default pool template is created at the time of Edge-LB installation. The template is rendered into a pool configuration based on the values of the app / task [labels](#supported-labels) (discussed below).
```